### ISSUE TITLE
Questions about the implemnetation of GPSConv in pyg

### ISSUE THREAD

**Issue by HelloWorldLTY** [pyg-team/pytorch_geometric#6817]
### 🐛 Describe the bug

![image](https://user-images.githubusercontent.com/43333475/221913130-4d187d61-bbe5-4132-85d5-3e470f647ccc.png)
Hi all, I am a little confused about the implementation of GPSConv in pyg library. I did not find the channel used for output embeddings. I notice that you choose to set the number of output channels the same as the input channels, so if I intend to perform information compression, how to achieve it? Thanks a lot.

### Environment

* PyG version: 2.2.0
* PyTorch version: 1.13.0
* OS: linux
* Python version:
* CUDA/cuDNN version:
* How you installed PyTorch and PyG (`conda`, `pip`, source):
* Any other relevant information (*e.g.*, version of `torch-scatter`):

**Comment by rusty1s** [comment]
That is correct. GPS assumes `in_channels == out_channels`. You can increase or decrease the feature size by simply appending or prepending a `Linear` layer that maps to your desired feature size.
